city_scrapers/spiders/il_lottery.py

```diff
@@ -0,0 +1,135 @@
+    timezone = "America/Chicago"
+    allowed_domains = ["www.illinoislottery.com"]
+    start_urls = ["https://www.illinoislottery.com/illinois-lottery/lottery-control-board/"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        if re.search(r'122 South Michigan Avenue, 19th Floor', response.text) is None:
+            raise ValueError('Meeting address has changed')
+
+        meetings = self._parse_upcoming(response)
+
+        for item in meetings:
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description='',
+                classification=BOARD,
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes='',
+                location={
+                    'name': 'Chicago Lottery Office',
+                    'address': '122 South Michigan Avenue, 19th Floor, Chicago, IL 60603'
+                },
+                source=response.url,
+            )
+            meeting['id'] = self._get_id(meeting)
+            meeting['status'] = self._get_status(meeting, text=item)
+            meeting['links'] = self._parse_agenda_links(response, date=meeting['start'].date())
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        suffix = ''
+        if 'QTR' in item:
+            suffix = " Quarterly Meeting"
+        elif 'Special' in item:
+            suffix = ' Special Meeting'
+        return "Lottery Control Board{}".format(suffix)
+
+    @staticmethod
+    def parse_time(source):
+        """Returns times given string with format h:mm am|pm"""
+        time_regex = re.compile(r'([1-9]):([0-5][0-9])\s?(am|pm)')
+        hour, minute, period = time_regex.search(source).groups()
+
+        hour = int(hour)
+        minute = int(minute)
+        if (period == 'pm') and (hour != 12):
+            hour += 12
+        return time(hour, minute)
+
+    @staticmethod
+    def parse_day(source):
+        """Returns date"""
+        # search for dates with '/' (ex: 08/16/19)
+        if "/" in source:
+            m = re.search(
+                r'.+[,-]\s(?P<month>\d{2})\/(?P<day>\d{2})\/(?P<year>\d{2})\s.+', source.strip()
+            )
+        # search for date in format "[month] [dd], [yyyy]" (ex: 'May 15, 2019')
+        else:
+            m = re.search(
+                r'.+[,-]\s*(?P<month>[A-Za-z]+)\s(?P<day>\d{1,2}),\s(?P<year>\d{4}).*',
+                source.strip()
+            )
+
+        dt = dateutil.parser.parse(
+            '{mo} {day} {year}'.format(
+                mo=m.group('month'), day=m.group('day'), year=m.group('year')
+            )
+        )
+        return dt.date()
+
+    def _parse_start(self, item):
+        """Parse start date and time."""
```

Comment:
It's up to you on this one because it seems like this is working, but we've usually been cleaning up date strings and then using `datetime.strptime` to do the parsing since the dates are usually a relatively standard format. If it seems like this fits more dates though it's fine to leave as-is

Reply:
Thanks for the tip! I updated parse_day() to use datetime.strptime